### DISCUSSION TITLE
Unlocking a user account on PhpMyAdmin

### DISCUSSION THREAD

**Discussion by Revanui** [leokhoa/laragon#787] [Q&A]

**Comment by iqbalnur21** [comment]
1. **Authentication Type Update**  
   Changed:
   ```php
   $cfg['Servers'][$i]['auth_type'] = 'cookie';
   ```
   To:
   ```php
   $cfg['Servers'][$i]['auth_type'] = 'config';
   ```
_file path: "C:\laragon\etc\apps\phpMyAdmin\config.inc.php"_

2. **Add User and Password Configuration**  
   Added the following lines if they don't already exist:
   ```php
   $cfg['Servers'][$i]['user'] = 'root';
   $cfg['Servers'][$i]['password'] = '';
   ```

3. **Restart Laragon**  
   Restarted Laragon to apply the configuration changes.

**Reply by Ringoinmel** [reply]
where is this file?  Thanks

**Reply by iqbalnur21** [reply]
my comment earlier is updated

**Reply by husnilkhatimi** [reply]
> _file path: "C:\laragon\etc\apps\phpMyAdmin\config.sample.inc.php"_

it should be this file right 'config.inc.php' ? not the sample config.

**Reply by iqbalnur21** [reply]
yes, right
